docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,37 +371,41 @@ which is a CDI bean and can be ``@Inject``ed.
 [[jdbcstore]]
 == Configure storing of Quarkus transaction logs in a database
 
-In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
+In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a Java Database Connectivity (JDBC) datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+However, in cloud-native apps, it is important to assess transactions after careful evaluation.
```

Comment:
I don't think this sentence is clear. What do you want to convey here?

Reply:
New sentence from @MichalMaler LGTM:

> However, in cloud-native apps, using an in-database transaction store has additional requirements.

All good for you @jmfinelli ?